packages/config/tests/parse/snapshots/tests.js.md

```diff
@@ -133,6 +133,345 @@ Generated by [AVA](https://avajs.dev).
     Could not parse configuration file␊
     Expected "/"", "/external/path", "/external/path"", "/external/path", "/external/path", "/external/path", "/external/path", "/external/path", "/external/path", "/external/path" or any character but end of input found.`
 
+## Configuration file - ambiguous backslash in TOML
+
+> Snapshot 1
+
+    `In netlify.toml, the following backslash should be escaped: /external/path␊
+    The following will be used instead: /external/path␊
```

Comment:
I assume there's no way of going around the normalisation for these paths here? 😕
Or maybe adding an extra programmatic assertion in our tests that validates the presence of the right strings?

Reply:
Fixed :+1:

Reply:
Awesome work with the tests 🙌